global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.4"
+    "version": "2.1.301"
   }
-}
\ No newline at end of file
+}

```

Comment:
My eyes! 😉

Reply:
Yeah. I don't love the flip-flopping, but eventually just became tired of fighting with the editors. And my .editorconfig says to add the final newline, so it _should_ stick.

Reply:
Oh! I was mistaken. I thought the change was the other way round. All good. Although of course, it should be a separate commit. 😛